src/views/OverviewPage.vue

```diff
@@ -55,26 +55,42 @@
         </section>
       </div>
     </div>
+
+    <div class="column">
+      <div class="card">
+        <h1 style="margin-left: 10px;">Recent Notes</h1>
```

Comment:
<img width="255" alt="截屏2020-04-24 下午8 12 00" src="https://user-images.githubusercontent.com/44103628/80269825-f5b11c80-8667-11ea-9d02-667783fbf826.png">
Avoid using custom stylesheets or it could lead to inconsistencies.